packages/x-adapter-platform/src/schemas/request/next-queries-request.schema.ts

```diff
@@ -0,0 +1,10 @@
+import { createMutableSchema, Schema } from '@empathyco/x-adapter-next';
+import { NextQueriesRequest } from '@empathyco/x-types';
+import { BasePlatformNextQueriesRequest } from '../../types/requests/next-queries-request.model';
+
+export const nextQueriesRequestSchema: Schema<NextQueriesRequest, BasePlatformNextQueriesRequest> =
```

Comment:
I didn't want to remove them in x-components because that would be coupling our platform API to the frontend. I think `x-components` can and should still be including those fields in the request because another API might be able to use them.

Regarding documentation, I agree that we can improve it. I'll document it in both `endpoint adapters`.